psutil/_psutil_posix.c

```diff
@@ -9,6 +9,7 @@
 #include <Python.h>
 #include <errno.h>
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
I think you can remove this, right?

Reply:
Good call, removed in 30825261446cadb71a7201e154bb40feee9eccd4